client/src/components/Backend/screens/UserContributions/UserContributions.tsx

```diff
@@ -84,7 +84,7 @@ const UserContributions = (props: Props) => {
       );
     }
     window.scrollTo(0, 0);
-  }, [dispatch]);
+  }, [dispatch, userStructure]);
```

Comment:
It seems that this one is causing an infinite loop which fetch data all the time (this is probably more a bug in the code of the effect)
You can experience it by going in the backoffice, in the "Fiches" tab

Reply:
I can't reproduce this 🤔 I checked the Network tab in developer tools and it just get's the data once (twice because running in dev mode actually)